hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -389,11 +393,13 @@ private Set<String> getDeltaFiles(OmSnapshot fromSnapshot,
       // End of Workaround
     }
 
-    if (deltaFiles.isEmpty()) {
+    if (useFullDiff || deltaFiles.isEmpty()) {
```

Comment:
This new condition looks reasonable to me.

But I'd like @sadanand48 or @hemantk-12 to review this change in this class as well.

Reply:
If useFullDiff is true then deltaFiles.isEmpty() is always true but I guess its ok to have this .